package.json

```diff
@@ -1,5 +1,7 @@
 {
   "dependencies": {
-    "express": "^4.18.1"
+    "express": "^4.18.1",
+    "react": "^18.2.0",
```

Comment:
Is this react stuff supposed to be here? This is the package.json for the api.

Reply:
I know React Select is required for the dropdown to work. I'll try deleting them and seeing if anything breaks.

Reply:
Deleted them and it still seems to work as intended. I repushed with those deleted now. 👍